Jenkinsfile-tag

```diff
@@ -0,0 +1,67 @@
+#!groovy
+
+def VERSION = ''
+
+pipeline {
+    agent any
+    stages {
+
+        stage('Checkout') {
+            steps {
+                checkout scm
+                sh 'git submodule update --init --recursive'
+                sh 'git rev-parse HEAD > git-sha.txt'
+                script {
+                    def baseVersionTag = readFile "VERSION"
+                    VERSION = baseVersionTag.trim();
+
+                    println "Version tag for this build is ${VERSION}"
+                }
+            }
+        }
+
+        stage('Deploy to RC') {
```

Comment:
Something I just realized is that these stages are gonna probably show up weird in the UI, but honestly nbd